config/manifests/gateway/istio/resources.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: inference-gateway
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+     kinds:
+     - kind: HTTPRoute
+     namespaces:
+       from: All
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: llm-route
```

Comment:
if we keep httproutes separate, should we align them with the same name? Alternatively, we can collapse them to one route shared between implementations (as discussed offline)